article/_posts/2019-05-09-CVAE.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: review
+title: "Learning Structured Output Representation using Deep Conditional Generative Models"
+tags: deep-learning VAE Segmentation 
```

Comment:
Not sure whether tags are case-insensitive.

Reply:
done